### PR TITLE
Adding Floyd Warshall Algo in C.

### PR DESCRIPTION
**Pull request by p-soham11**
Fixes #2205 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Adding Floyd Warshall Algorithm in C. 

#### Languages Used:
- C

#### Files Added:
- Floyd_Warshall_Algorithm.c

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            else
+                printf ("%7d", dist[i][j]);
+        }
+        printf("\n");
+    }
+}
+
+int main()
+{
+    int graph[A][A] = {{0, 9, INF, 8},
```
Comment (singh-shreya6) [inline Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c:64]: Please take dynamic inputs.

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c [hunk 2]
```diff
@@ -0,0 +1,95 @@
+//Floyd Warshall Algorithm using C
+
+#include<stdio.h>
+
+//Vertices count in the Graph
+#define A 4
+
+//Defining INF(infinite) as a large value.
+//It's used for non-connected vertices
+#define INF 999999
```
Comment (singh-shreya6) [inline Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c:10]: use INT_MAX for infinity

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c [hunk 3]
```diff
@@ -0,0 +1,95 @@
+//Floyd Warshall Algorithm using C
+
+#include<stdio.h>
+
+//Vertices count in the Graph
+#define A 4
```
Comment (singh-shreya6) [inline Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c:6]: Please don't take hard coded value. Take inputs dynamically.

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by p-soham11** [comment]
Yes, Mam I was trying to learn about solving the problem by using dynamic input.

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c (hunk 2 above)
Comment (p-soham11) [reply Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c:10]: Ma'am, I previously used macro, but one of the mentor told not to use and I was having squash problems.
So, shall I change it to macro ?

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c (hunk 1 above)
Comment (p-soham11) [reply Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c:64]: Ma'am, I was trying to figure out the solution with dynamic input.
But unfortunately it is becoming really hard for me to work with it.

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by singh-shreya6** [comment]
So many days and the changes have not been made. Please raise a new PR with suggested changes if you plan to work on this. Closing this one.

**Pull request closed without merging**